app/forms/h2_form_builder.rb

```diff
@@ -0,0 +1,42 @@
+class H2FormBuilder < ActionView::Helpers::FormBuilder
```

Comment:
As part of abstracting nested forms, helpers for nested form buttons are provided. These provide the necessary data attributes for the nested form stimulus controller, so as to avoid have them copied throughout the ERBs.